src/main/java/com/minecolonies/inventory/GuiField.java

```diff
@@ -0,0 +1,83 @@
+package com.minecolonies.inventory;
+
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.entity.ai.citizen.farmer.Field;
+import com.minecolonies.lib.Constants;
+import com.minecolonies.tileentities.ScarecrowTileEntity;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.client.renderer.GlStateManager;
+import net.minecraft.entity.player.InventoryPlayer;
+import net.minecraft.util.BlockPos;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.World;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+/**
+ * Class which creates the GUI of our field inventory.
+ */
+@SideOnly(Side.CLIENT)
+public class GuiField extends GuiContainer
+{
+    /**
+     * The resource location of the GUI background.
+     */
+    private static final ResourceLocation TEXTURE = new ResourceLocation(Constants.MOD_ID, "textures/gui/scarecrow.png");
+
+    /**
+     * The X-OFFSET of the string in the GUI.
+     */
+    private static final int X_OFFSET = 8;
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)